hw1.xml

```diff
@@ -0,0 +1,617 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<con:soapui-project id="72ebc347-87e1-42bc-a573-65ecba82b485" activeEnvironment="Default" name="lab1.yandex.speller" soapui-version="5.6.0" abortOnError="false" runType="SEQUENTIAL" resourceRoot="" xmlns:con="http://eviware.com/soapui/config"><con:settings><con:setting id="com.eviware.soapui.impl.wsdl.actions.iface.tools.soapui.TestRunnerAction@values-local"><![CDATA[<xml-fragment xmlns:con="http://eviware.com/soapui/config">
```

Comment:
work is needed  - see details in comments.

Reply:
done